blockchain/blockchain-wallet-v4-frontend
# Feature/pl-bug-fixes

## Description
585, 550, 421

## Change Type
Please enter one or more of the following: 
- Bug Fixes

## Testing Steps
585: change bch wallet label and x out of input
550: archive and unarchive btc wallet
421: see that security center is not broken on safari

## PR Creator Checklist

## PR Reviewer Checklist
